blog/2024-07-10-release-notes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Luigi v2.14  
+seoMetaDescription: Release notes for Luigi v2.14
+author:
+  - Mahati Shankar
+layout: blog
+---
+
+You can read about the new features in Luigi v2.13 in the release notes below.
+
+<!-- Excerpt -->
+
+
+#### E2E Tests With Nightwatch, WebdriverIO and Puppeteer
+
+You can now use frameworks instead of Cypress which has been deprecated.
+
+#### getCurrentRoute Has Been Added
+
+getCurrentRoute has been added to the Luigi core API which as the name suggests returns the current Luigi route.
+
+#### getCurrentTheme in WC Client
+
+The method is now part of the core API so you can now use it in the WebComponent client too.
+
+#### Fiddle Theming Example
+
+To see the example in action, open [Luigi Fiddle](https://fiddle.luigi-project.io/) then choose the user icon at the top right and finally select `Settings` from the popover.
```

Comment:
```suggestion

```
Please remove it, because the blog post is only about Luigi core/client/container. (my mistake)